x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/add_inference_pipeline_flyout.test.tsx

```diff
@@ -225,24 +214,6 @@ describe('AddInferencePipelineFlyout', () => {
       });
       testStepStatus(FIELDS_STEP_INDEX, 'Fields', 'complete');
     });
-    it('mappings step is current when on step', () => {
-      setMockValues({
-        ...DEFAULT_VALUES,
-        addInferencePipelineModal: {
-          step: AddInferencePipelineSteps.Mappings,
-        },
-      });
-      testStepStatus(MAPPINGS_STEP_INDEX, 'Mappings', 'current');
```

Comment:
Please remove `MAPPINGS_STEP_INDEX` and update the ordinals of the other `_INDEX` constants.

Reply:
Done, thanks for the catch!